tests/queries/0_stateless/01700_deltasum.sql

```diff
@@ -0,0 +1,7 @@
+select deltaSum(arrayJoin([1, 2, 3]));
```

Comment:
It seems like this function support floats, decimals as well consider adding some test cases for them.